### PR TITLE
update schemas

### PR DESCRIPTION
**Pull request by KamilSzostak**
#327

### CODE REVIEW

File: JavaScript/JavaScriptSDK.Interfaces/Contracts/Generated/RemoteDependencyData.ts [hunk 1]
```diff
@@ -1,51 +1,99 @@
...
         public min: number;
         public max: number;
         public stdDev: number;
         public dependencyKind: AI.DependencyKind;
+        
+        /**
+         * Indication of successfull or unsuccessfull call.
+         */
+        public success: boolean;
         public async: boolean;
```
Comment (SergeyKanzhelev) [inline JavaScript/JavaScriptSDK.Interfaces/Contracts/Generated/RemoteDependencyData.ts]: isn't async is gone in public schema?

File: JavaScript/JavaScriptSDK.Interfaces/Contracts/Generated/RemoteDependencyData.ts [hunk 2]
```diff
@@ -1,51 +1,99 @@
...
-
-        /* Following fields should be marked as depricated in the next release, and must be removed in the subsequent breaking release */
         public kind: AI.DataPointType;
         public value: number;
+        
+        /**
+         * Request duration in format: DD.HH:MM:SS.MMMMMM. Must be less than 1000 days.
+         */
+        public duration: string;
         public count: number;
```
Comment (SergeyKanzhelev) [inline JavaScript/JavaScriptSDK.Interfaces/Contracts/Generated/RemoteDependencyData.ts]: these fields gone in public schema

Comment (KamilSzostak) [reply JavaScript/JavaScriptSDK.Interfaces/Contracts/Generated/RemoteDependencyData.ts]: did we replace duration with 'value'? Or it's just gone and the JS SDK should not report any duration?

File: JavaScript/JavaScriptSDK.Interfaces/Contracts/Generated/RemoteDependencyData.ts (hunk 1 above)
Comment (KamilSzostak) [reply JavaScript/JavaScriptSDK.Interfaces/Contracts/Generated/RemoteDependencyData.ts]: Ok, I'll remove it.

File: JavaScript/JavaScriptSDK.Interfaces/Contracts/Generated/RemoteDependencyData.ts (hunk 2 above)
Comment (KamilSzostak) [reply JavaScript/JavaScriptSDK.Interfaces/Contracts/Generated/RemoteDependencyData.ts]: it's there: https://github.com/Microsoft/ApplicationInsights-Home/blob/master/EndpointSpecs/Schemas/Bond/RemoteDependencyData.bond

Comment (SergeyKanzhelev) [reply JavaScript/JavaScriptSDK.Interfaces/Contracts/Generated/RemoteDependencyData.ts]: I meant `count`. It's not there

Comment (KamilSzostak) [reply JavaScript/JavaScriptSDK.Interfaces/Contracts/Generated/RemoteDependencyData.ts]: Yes, I reviewed all bond files in AppInsights-Home and remove all fields that are not present there. 

After removing those fields I've noticed some no longer used code, which I'm removing right now.

**Pull request merged**